### PR TITLE
Create qiime2_docker.md

### PR DESCRIPTION
**Pull request by evezeyl**
added tutorial for working with qiime2 from container

### CODE REVIEW

File: docs/source/tutorials/qiime2_docker.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Getting Qiime 2024.5 via container
+
+## Setting up your `.bashrc` config for Apptainer
+
+[Apptainer](https://apptainer.org/) is the system on SAGA that allows you to use containers.
+
+Some configuration is necessary to make it easier to use containers
```
Comment (Thomieh73) [inline docs/source/tutorials/qiime2_docker.md:7]: add a dot to the end of the line

Comment (Thomieh73) [inline docs/source/tutorials/qiime2_docker.md:7]: Also add the sentence: "To do that we need to modify the file `.bashrc`, which is located in your home directory.

File: docs/source/tutorials/qiime2_docker.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Copy those lines and put them into your .bashrc
+Replace what is in between `<username>` with your own username
```
Comment (Thomieh73) [inline docs/source/tutorials/qiime2_docker.md:10]: Move lines 9 and 10 to below the instructions on how to access the bashrc file.

**Review by Thomieh73 — CHANGES_REQUESTED** [review]
Hi Eve, I have added a few small notes to this document, so that you can change it.

File: docs/source/tutorials/qiime2_docker.md (hunk 1 above)
Comment (evezeyl) [reply docs/source/tutorials/qiime2_docker.md]: ok

**Review by evezeyl — COMMENTED** [review]
should be ok

**Comment by evezeyl** [comment]
Hi, I made the changes + some more - but do not find way to say its done

**Review by Thomieh73 — APPROVED** [review]
This looks good to me

**Comment by Thomieh73** [comment]
Hi @evezeyl  I have approved the changes and done a pull request and merged the branches.

**Pull request merged**